eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqlitePaymentsDb.scala

```diff
@@ -180,7 +181,7 @@ class SqlitePaymentsDb(sqlite: Connection) extends PaymentsDb with Logging {
     using(sqlite.prepareStatement(queryStmt)) { statement =>
       statement.setLong(1, from)
       statement.setLong(2, to)
-      if (pendingOnly) statement.setLong(3, Instant.now().getEpochSecond)
+      if (pendingOnly) statement.setLong(3, Platform.currentTime.milliseconds.toSeconds)
```

Comment:
Shouldn't that be `Platform.currentTime.milliseconds.toMillis`?

Reply:
No because Payment Request is a protocol object and use seconds, we also store "expiry" in seconds and the "expire_at" field of the payment db uses seconds as time unit [here](https://github.com/ACINQ/eclair/blob/master/eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqlitePaymentsDb.scala#L139).